### ISSUE TITLE
Executing randomly generated cart crashes Click4

### ISSUE THREAD

**Issue by tobiasvl** [josefnpat/click4#3]
When loading a cart without having saved anything first, the memory will be filled with random nibbles. I assume this is intentional, and it is charming. However, when trying to run this newly loaded gibberish, Click4 will crash:

![image](https://user-images.githubusercontent.com/80824/68517056-39fd5b80-0286-11ea-827c-10d3410a65ad.png)

Saving the cart (and optionally reloading it, just to make sure) and then executing it will still elicit a crash. However, if you quit Click4, start it again, load the previously saved cart, and _then_ run it, it will run fine. So the problem isn't with the cart itself.

So in summary, to reproduce this bug:

* Start Click4 and delete any saved cart
* Load the non-existent cart and see the screen fill up with random colors
* Execute this cart and Click4 will crash

And to reproduce the non-crash:

* Start Click4 and delete any saved cart
* Load the non-existent cart and see the screen fill up with random colors
* Save the cart
* Quit Click4
* Start Click4
* Load cart
* Execute this cart and everything works fine

By the way, this is one of the best ideas for a fantasy console I've seen. Truly inspiring!

**Comment by josefnpat** [comment]
@tobiasvl Thank you for the kind words, and thank you for such a detailed bug report! I will look into this as soon as I have time, thank you!
